prost-build/Cargo.toml

```
@@ -23,7 +23,7 @@ cleanup-markdown = ["pulldown-cmark", "pulldown-cmark-to-cmark"]
 [dependencies]
 bytes = { version = "1", default-features = false }
 heck = "0.4"
-itertools = { version = ">=0.10, <0.12", default-features = false, features = ["use_alloc"] }
+itertools = { version = ">=0.10, <0.13", default-features = false, features = ["use_alloc"] }
```

Personally, I think this syntax explains better the intent of the version bounds.
```suggestion
itertools = { version = ">=0.10, <=0.12", default-features = false, features = ["use_alloc"] }
```

---

Wouldn't this disallow e.g. `0.12.1`?

---

> Wouldn't this disallow e.g. `0.12.1`?

Have you tested that? I believe my suggestion is equivalent.

```
$ cargo update
    Updating crates.io index
    Updating itertools v0.11.0 -> v0.12.1
```

---

You are right, sorry for the noise.